### PR TITLE
Update filter tooltip messages for 4.3

### PR DESCRIPTION
**Pull request by ctzsnooze**
Updates the filter messages for the help icons in 4.3

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (Asizon) [inline locales/en/messages.json:3899]: It is wrong since we are not using the long name in these cases, please remove "Gyro Lowpass 1"
Maybe we should name this "Static frequency [Hz] ??

File: locales/en/messages.json
Comment (Asizon) [inline locales/en/messages.json:3922]: Same here

File: locales/en/messages.json
Comment (Asizon) [inline locales/en/messages.json:3926]: Same here

File: locales/en/messages.json
Comment (Asizon) [inline locales/en/messages.json:4073]: Same here

**Review by asizon — CHANGES_REQUESTED** [review]

File: locales/en/messages.json
Comment (ctzsnooze) [reply locales/en/messages.json]: Thank you!

So... I removed the above message definitions, and in html, applied `pidTuningCutoffFrequency` in place of  `pidTuningGyroLowpassFrequency `.  It now says, "Static cutoff [Hz]", because it is static.  I named all such static cutoffs the same way.

The original problem was that someone removed the name `pidTuningGyroLowpassFrequency ` from locales, but didn't remove it from the html, so it was throwing an error.

**Comment by ctzsnooze** [comment]
Thanks @Asizon - now all static cutoffs have the label `Static cutoff [Hz]`, like this:

Screen Shot 2022-01-10 at 08 54 10

File: locales/en/messages.json
Comment (Asizon) [reply locales/en/messages.json]: Ouch, maybe i have removed it. Sorry

File: locales/en/messages.json
Comment (Asizon) [inline locales/en/messages.json:3960]: Overloocked this indendation.

**Review by asizon — CHANGES_REQUESTED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (ctzsnooze) [reply locales/en/messages.json]: thanks !  fixed.

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by asizon — APPROVED** [review]

**Pull request merged**
